packages/core-js/internals/fix-regexp-well-known-symbol-logic.js

```diff
@@ -75,7 +84,11 @@ module.exports = function (KEY, length, exec, sham) {
   if (
     !DELEGATES_TO_SYMBOL ||
     !DELEGATES_TO_EXEC ||
-    (KEY === 'replace' && !(REPLACE_SUPPORTS_NAMED_GROUPS && REPLACE_KEEPS_$0)) ||
+    (KEY === 'replace' && !(
+      REPLACE_SUPPORTS_NAMED_GROUPS &&
+      REPLACE_KEEPS_$0 &&
+      REGEXP_REPLACE_SUBSTITUTES_UNDEFINED_CAPTURE
```

Comment:
Other criteria mean that the implementation is correct, but `REGEXP_REPLACE_SUBSTITUTES_UNDEFINED_CAPTURE` that it's buggy, so seem it should be `!REGEXP_REPLACE_SUBSTITUTES_UNDEFINED_CAPTURE`.